src/main/java/com/epam/aidial/core/controller/FileMetadataController.java

```diff
@@ -21,7 +21,7 @@ protected Future<?> handle(ResourceDescription resource) {
         BlobStorage storage = proxy.getStorage();
```

Comment:
Are you going to change metadata API in this pr?

Reply:
I don't have other resources to deal with right now. Let's implement is later

Reply:
The frontend will be able to migrate on these change as soon as possible.

Reply:
I think I misunderstood your point. We should fix metadata routing in this PR